janusgraph-berkeleyje/src/test/resources/release-test-groups/bdb1

```diff
@@ -0,0 +1 @@
+org.apache.tinkerpop.gremlin.process.TraversalPerformanceTest
```

Comment:
Of course, depending on the granularity of division, we may need a tool to make sure that TP updates and janusgraph-test updates did not introduce new tests that are not labelled with the new test categories.